tests/metrics/network/network-metrics-memory-1g.sh.in

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Description:
+#  Measures Proportional Set Size memory while running an
+#  inter (docker<->docker) 1Gbps network bandwidth using nuttcp
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+
+source "${SCRIPT_PATH}/../../lib/test-common.bash"
+
+# Currently default nuttcp has a bug
+# see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=745051
+# Image name
+image=gabyct/nuttcp
+# This is required in order to reduce standard deviation
+total_time=6
+# This time (seconds) is required when
+# server and client are more stable, we need to
+# have server and client running for sometime and we
+# need to avoid to measure at the beginning of the running
+middle_time=3
+
+# Rate limit (speed at which transmitter send data, megabytes)
+rate_limit=10000
+
+function setup {
+	runtime_docker
+	kill_processes_before_start
+}
+
+# This script will perform all the measurements using a local setup
+
+function start_client_server {
+	setup
+	total_memory=$(mktemp)
+	server_name="nuttcp-server"
+	client_name="nuttcp-client"
+
+	server_command="/root/nuttcp -S"
+	$DOCKER_EXE run -tid --name=${server_name} ${image} bash > /dev/null
+        server_address=$($DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" ${server_name})
+
+        client_command="/root/nuttcp -R${rate_limit}m -T${total_time} ${server_address}"
+	$DOCKER_EXE exec ${server_name} bash -c "${server_command}"
+	$DOCKER_EXE run -d --name=${client_name} ${image} bash -c "${client_command}" > /dev/null
+
+	# Measurement after client and server are more stable
+	echo >&2 "WARNING: sleeping for $middle_time seconds in order to have server and client stable"
+	sleep ${middle_time}
+	
+	${memory_command} -P @QEMU_PATH@ | tail -n 2 > "$total_memory"
```

Comment:
I think if you add the --no-header here (as used at this link), you can drop the `head` maybe:
https://github.com/01org/cc-oci-runtime/blob/master/tests/metrics/density/docker_memory_usage.sh.in#L70
Also, I'm not convinced about the split of the smem options between this line and the pss_memory() function setting up the memory_command variable - it feels a bit 'fragmented', particularly as there is only a single use in the file. If you can think of a way to make that cleaner, please do.